mache-couchbase/src/main/java/com/excelian/mache/couchbase/builder/CouchbaseProvisioner.java

```diff
@@ -21,18 +22,17 @@
 public class CouchbaseProvisioner implements StorageProvisioner {
 
     private final BucketSettings bucketSettings;
-    private CouchbaseEnvironment couchbaseEnvironment;
-    private List<String> nodes;
+    private Cluster cluster;
+
     private String adminUser;
     private String adminPassword;
     private SchemaOptions schemaOptions;
 
-    private CouchbaseProvisioner(CouchbaseEnvironment couchbaseEnvironment, BucketSettings bucketSettings,
-                                 List<String> nodes, String adminUser, String adminPassword,
+    private CouchbaseProvisioner(Cluster cluster, BucketSettings bucketSettings,
+                                 String adminUser, String adminPassword,
                                  SchemaOptions schemaOptions) {
-        this.couchbaseEnvironment = couchbaseEnvironment;
+        this.cluster = cluster;
```

Comment:
Does this work with no clustering options?....Or does Couch always have a cluster, even if there's one node?